src/AccessibilityInsights.AutomationTests/AutomationSessionUnitTests.cs

```diff
@@ -57,22 +62,19 @@ public void NewInstance_NoInstanceExists_ExceptionInCtor_RethrowsSameException()
                 }
             }
         }
+#endif
 
         [TestMethod]
         [Timeout (2000)]
         public void NewInstance_ClearInstance_NotPowerShell_AssemblyResolverIsNull_DoesNotThrow()
         {
-            using (ShimsContext.Create())
+            try
             {
-                InitializeShims();
```

Comment:
>InitializeShims [](start = 16, length = 15)

These should have been removed with commit 523568c0919c30fd2e19d6b3b83804be9ad88a0e. Thanks for removing them here